sswf-core/src/it/scala/com/bazaarvoice/sswf/ListOpenExecutionTest.scala

```diff
@@ -54,6 +53,8 @@ class ListOpenExecutionTest extends FlatSpec {
     try {
       assert(waitForStepResult().getResult === StepResult.serialize(Success(None)))
 
+      Thread.sleep(1000);
```

Comment:
Ah. Ew, that's lame. I'm really not a fan of sleeping in tests because it's unreliable. A more robust way would be to convert the asynchronous calls to synchronous, either by e.g. waiting on Futures or in the case of distributed systems, polling. I realize the value proposition probably doesn't appear to be there, but

Reply:
Well I can add a comment here describing why it is in there.

Reply:
I meant to do that and forgot, took me a while to figure out what was going on here.

Reply:
Okay.